flatsurf/geometry/gl2r_orbit_closure.py

```diff
@@ -755,20 +755,20 @@ def lift(self, v):
 
             sage: T = polygons.triangle(3,4,13)
             sage: S = similarity_surfaces.billiard(T)
-            sage: S = S.minimal_cover("translation").erase_marked_points() # optional: pyflatsurf
-            sage: O = GL2ROrbitClosure(S)  # optional: pyflatsurf
-            sage: for d in O.decompositions(4, 20):  # optional: pyflatsurf
+            sage: S = S.minimal_cover("translation").erase_marked_points() # long time (3s), optional: pyflatsurf
+            sage: O = GL2ROrbitClosure(S)  # long time (above), optional: pyflatsurf
+            sage: for d in O.decompositions(4, 20):  # long time (2s), optional: pyflatsurf
```

Comment:
Here I would say that it is suspicious dependsing on how many `d` are involved in this loop.

Reply:
This is now #124.